tools/bootnode/BUILD.bazel

```diff
@@ -80,3 +55,15 @@ go_test(
         "@com_github_sirupsen_logrus//:go_default_library",
     ],
 )
+
+prysm_image_upload(
+    name = "push_images",
+    binary = ":bootnode",
+    entrypoint = ["/bootnode"],
```

Comment:
does this thing still need visibility? `visibility = ["//visibility:private"],`

Reply:
No, that isn't necessary for this macro